src/main/java/com/conveyal/datatools/manager/jobs/NotifyUsersForSubscriptionJob.java

```diff
@@ -27,32 +27,19 @@
     private static final String APPLICATION_NAME = DataManager.getConfigPropertyAsText("application.title");
     private static final String APPLICATION_URL = DataManager.getConfigPropertyAsText("application.public_url");
 
-    private NotifyUsersForSubscriptionJob(String subscriptionType, String target, String message) {
+    public NotifyUsersForSubscriptionJob(String subscriptionType, String target, String message) {
+        super();
         this.subscriptionType = subscriptionType;
         this.target = target;
         this.message = message;
     }
 
-    /**
-     * Convenience method to create and schedule a notification job to notify subscribed users.
-     */
-    public static void createNotification(String subscriptionType, String target, String message) {
+    @Override
+    public void jobLogic () {
         if (APPLICATION_URL == null || !(APPLICATION_URL.startsWith("https://") || APPLICATION_URL.startsWith("http://"))) {
             LOG.error("application.public_url (value={}) property must be set to a valid URL in order to send notifications to users.", APPLICATION_URL);
             return;
         }
-        NotifyUsersForSubscriptionJob notifyJob = new NotifyUsersForSubscriptionJob(subscriptionType, target, message);
-        DataManager.lightExecutor.execute(notifyJob);
-        LOG.info("Notification job scheduled in light executor");
-    }
-
-    @Override
-    public void run() {
-        notifyUsersForSubscription();
-    }
-
-    // TODO: modify method so that it receives both a feed param and a updateFor param?
-    private void notifyUsersForSubscription() {
```

Comment:
Please revert due to reasons list in https://github.com/ibi-group/datatools-server/pull/348#discussion_r512341045.

Reply:
Updated.